htdocs/expedition/card.php

```diff
@@ -1199,9 +1199,9 @@
 
 		if ($totalWeight > 0)
 		{
-			if (!empty($object->trueWeight)) print ' ('.$langs->trans("SumOfProductWeights").': ';
+			if (!empty($totalWeight)) print ' ('.$langs->trans("SumOfProductWeights").': ';
```

Comment:
Old code looks good.
If $object->trueWeight is set (so a forced value of weight was set at the shipment level), then we must add a label on a second weight (sum of all products) to explain difference between 2 weight.
If no weight is forced, we don't need this explanation